dashbuilder-website-docs/src/main/asciidoc/guides/yaml_guide.adoc

```diff
@@ -665,6 +831,16 @@ pages:
 .Table with order configuration
 image::guides/yaml/orderingTable.png[Table with order configuration]
 
+Displayers have the following new fields:
```

Comment:
Please avoid using *new*, our documentation does not have version, so new may sound weird for users